commcare_connect/opportunity/views.py

```
@@ -2296,78 +2313,43 @@ def get_table(self, **kwargs):
+            "pending": _("Pending NM Review"),
+            "pending_review": _("Pending PM Review"),
+            "agree": _("Agree") if self.request.is_opportunity_pm else _("Approved"),
+            "disagree": _("Disagree") if self.request.is_opportunity_pm else _("Revalidate"),
+        }
+        tabs_to_display = self.get_tabs()
+        tabs = []
+        for tab in tabs_to_display:
+            tabs.append({"name": tab, "label": tabs_to_labels[tab], "count": user_visit_counts.get(tab, 0)})
+        return tabs
+
+    @functools.cache
```

Ruff (0.15.12)

[warning] Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)